pkg/api/server/register_containers.go

```diff
@@ -716,7 +716,7 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/libpod/containers/{name}/json"), APIHandler(s.Context, libpod.GetContainer)).Methods(http.MethodGet)
-	// swagger:operation POST /libpod/containers/{name}/kill libpod libpodKillContainer
+	// swagger:operation GET /libpod/containers/{name}/kill libpod libpodKillContainer
```

Comment:
kill should be a POST -> https://docs.docker.com/engine/api/v1.40/#operation/ContainerKill

Reply:
The function declaration must be fixed, then:
```
r.HandleFunc(VersionedPath("/libpod/containers/{name}/kill"), APIHandler(s.Context, libpod.KillContainer)).Methods(http.MethodGet)
```